presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -112,15 +112,22 @@ catalog, and deny all other access, you can use the following rules:
       ]
     }
 
-Username Extraction
--------------------
+Principal Rules
+---------------
 
 These rules serve to enforce a specific matching between a principal and a
-specified user name. It consists of a list of regular expressions which will
-perform group extraction and compare it with the provided user name.
-If a provided expression extracts the same user name, the user will have
-access to the database.
-If no expression are specified, no checks will be performed.
+specified user name. The principal is granted authorization as a user based
+on the first matching rule read from top to bottom. If no rules are specified,
+no checks will be performed. If no rule matches, user authorization is denied.
+Each rule is composed of the following fields:
+
+* ``principal`` (required): regex to match and group against a principal.
+* ``user`` (optional): replacement value to substitute against principal name.
+  The result of substitution will be used as the regex to match the user name.
```

Comment:
> match

in equality sense?

Reply:
ah, i missed "used as the regex" part, sorry
 (should be "used as _a_ regex" btw)